netket/experimental/operator/_fermion_operator_2nd.py

```diff
@@ -231,9 +241,37 @@ def from_openfermion(
     def __repr__(self):
         return (
             f"FermionOperator2nd(hilbert={self.hilbert}, "
-            f"n_operators={len(self._operators)}, dtype={self.dtype})"
+            f"n_operators={len(self.terms)}, dtype={self.dtype})"
         )
 
+    @property
+    def terms(self) -> OperatorTermsList:
+        return self._terms
+
+    @property
+    def weights(self) -> OperatorWeightsList:
+        return self._weights
+
+    @property
+    def constant(self) -> Union[float, complex]:
+        return self._constant
+
+    @property
+    def operator_dict(self) -> dict[OperatorTerm, Union[float, complex]]:
+        """
+        Return a dict with terms as keys and weights as values.
+        This does not include the constant term.
+        """
+        return dict(zip(self.terms, self.weights))
+
+    @classmethod
+    def identity(cls, hilbert: AbstractHilbert, **kwargs):
+        return cls(hilbert, [], [], constant=1.0, **kwargs)
+
+    @classmethod
+    def zero(cls, hilbert: AbstractHilbert, **kwargs):
+        return cls(hilbert, [], [], constant=0.0, **kwargs)
+
```

Comment:
in the spirit of 'one way to do it' I would not add those, as they are already inside of fermion.py and this is duplicating functionality.